src/Contracts/Types.d.ts

```diff
@@ -0,0 +1,25 @@
+/*
+    Copyright (c) Microsoft Corporation. All rights reserved.
+    Licensed under the MIT license.
+*/
+
+export interface IImageDetails {
+    imageName: string;
+    imageUri: string;
+    hash: string;
+    baseImageName: string;
+    distance?: number;
+    imageType: string;
+    mediaType: string;
+    tags: Array<string>;
+    layerInfo: Array<IImageLayer>;
+    buildId: number;
```

Comment:
reword buildId to runId, buildVersion to pipelineVersion, buildDefinitionName to pipelineName, buildDefinitionId to pipelineId

Reply:
done